climpred/metrics.py

```diff
@@ -147,9 +89,26 @@ def get_metric_function(metric):
             metric = _nmae
         elif metric.lower() == 'uacc':
             metric = _uacc
+        elif metric.lower() == 'msss_murphy':
+            metric = _msss_murphy
+        elif metric.lower() in ['c_b', 'conditional_bias']:
+            metric = _conditional_bias
+        elif metric.lower() in ['u_b', 'unconditional_bias', 'bias']:
+            metric = _bias
+        elif metric.lower() == 'std_ratio':
+            metric = _std_ratio
+        elif metric.lower() == 'bias_slope':
+            metric = _bias_slope
+        elif metric.lower() == 'less':
+            metric = _less
+        elif metric.lower() == 'crps':
+            metric = _crps
+        elif metric.lower() == 'crpss':
+            metric = _crpss
```

Comment:
Another way of writing a bunch of elif statements is using a dict:

```
METRIC_DICT = {
    'bias': _bias,
    'u_b': _bias,
    'less': _less,
    'crps': _crps,
...
}

metric = METRIC_DICT[metric.lower()]
```

Reply:
I also think we should change this.

Reply:
the only challenge I see now is that we have a few names for some metrics and now they all run in the tests. Maybe we can define a second metric dict which only contains names which correspond to unique metric functions. To be done later.

Reply:
got rid of .lower. users should be able to get metrics right or face an error.